### ISSUE TITLE
Problems with the web panel

### ISSUE THREAD

**Issue by notevil1337** [UnamSanctam/UnamWebPanel#284]
Hello Unam again, I have some problem with the panel again, I just downloaded the last web panel from your link and when I uploaded the files it doesn't show me the statistics page anymore (only loading and loading) (it's less important) and the second problem I got is 0 Hashrate on the web panel, when the pool is 2.0 kh/s since miner starts, but I got 0s on the webpanel

**Comment by UnamSanctam** [comment]
Hmm, the hashrate reporting is directly from the miner so that should always be correct unless there's something weird in the endpoint. And the statistics should be working too, can you check the `__UNAM_LIB\Logs` folder and see if there's anything there?

**Comment by notevil1337** [comment]
@UnamSanctam I fixed statistics but its still like this but sometimes it says Active with 0 hashrate sometimes it says Stopped
![image](https://github.com/UnamSanctam/UnamWebPanel/assets/141959915/3ef0f216-55b3-4ce6-9464-d68a1b38505e)

**Comment by notevil1337** [comment]
Maybe GPU does not have enough resources to mine ?

**Comment by UnamSanctam** [comment]
Yes that almost always means that the user did something that starts using a lot of the GPU.

**Comment by notevil1337** [comment]
> Yes that almost always means that the user did something that starts using a lot of the GPU.

I mean I think that GPU have only 4GB, should it work? or it needs at leaste 5GB?

**Comment by UnamSanctam** [comment]
4GB can mine some cryptocurrencies (as long as the cryptocurrency DAG size is below that), but what really matters is how much other things are using.

**Comment by notevil1337** [comment]
![image](https://github.com/UnamSanctam/UnamWebPanel/assets/141959915/9f9f4eb1-0583-4a67-b376-e985fe352932)
It would be a nice update to add what is taking resources from the GPU @UnamSanctam

**Comment by UnamSanctam** [comment]
Unfortunately it 's not possible to know that, there's no way to know what is using GPU resources.
